### PR TITLE
chore(fmg): marker gene calculation only uses healthy cells

### PR DESCRIPTION
**Pull request by atarashansky**
- #3803

### CODE REVIEW

File: backend/wmg/pipeline/summary_cubes/marker_genes.py [hunk 1]
```diff
@@ -52,10 +52,12 @@ def create_marker_genes_cube(corpus_path: str):
                 "tissue_ontology_term_ids": [tiss],
                 "cell_type_ontology_term_ids": [ct],
                 "organism_ontology_term_id": organism,
+                "disease_ontology_term_ids": ["PATO:0000461"],
```
Comment (atarashansky) [inline backend/wmg/pipeline/summary_cubes/marker_genes.py:55]: PAT0:0000461 is the ontology term ID for `normal` cells.

File: backend/wmg/data/query.py [hunk 2]
```diff
@@ -169,14 +169,10 @@ def retrieve_top_n_markers(query_result, test, n_markers):
     attrs = [f"p_value_{test}", f"effect_size_{test}"]
     col_names = ["p_value", "effect_size"]
     markers = query_result[["gene_ontology_term_id"] + attrs].rename(columns=dict(zip(attrs, col_names)))
+    markers = markers[markers["effect_size"].astype("str") != "nan"]
```
[the remark this answers is not in the document]
Comment (atolopko-czi) [reply backend/wmg/data/query.py:171]: Why filter nan as a string as opposed to, say, `notna()`? If there's a good reason, worth capturing as a comment.

File: backend/wmg/pipeline/summary_cubes/marker_genes.py (hunk 1 above)
Comment (atolopko-czi) [reply backend/wmg/pipeline/summary_cubes/marker_genes.py:55]: worth making that clear via an appropriately named constant

**Review by atolopko-czi — APPROVED** [review]
Looks reasonable, with a couple nits.

**Pull request merged**